sdk/python/feast/templates/local/feature_repo/example_repo.py

```diff
@@ -88,6 +90,9 @@ def transformed_conv_rate(inputs: pd.DataFrame) -> pd.DataFrame:
         driver_stats_fv[["conv_rate"]],  # Sub-selects a feature from a feature view
         transformed_conv_rate,  # Selects all features from the feature view
     ],
+    logging_config=LoggingConfig(
```

Comment:
This is affecting all the new repos created with `feast init`, right?
If this is the case please add a note to the PR so that we can alert the community reviewers